BackEnd/src/main/kotlin/quest/marketstack/tradingapp/domain/opentrades/MongoOpenTradeDataSource.kt

```diff
@@ -0,0 +1,47 @@
+package quest.marketstack.tradingapp.domain.opentrades
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.context.annotation.Primary
+import org.springframework.context.annotation.Profile
+import org.springframework.stereotype.Component
+import quest.marketstack.tradingapp.model.OpenTrade
+import quest.marketstack.tradingapp.model.TradeExec
+
+@Primary
+@Profile("!test")
```

Comment:
```suggestion
```
This shouldn't be necessary to include. If you remove this line and your code isn't working, that's an issue with the code, or maybe how you structured your files or something. Remove this, and if it doesn't work, try to find a solution that doesn't involve including this line. If you can't figure it out we can work together